packages/mermaid/src/schemas/config.schema.yaml

```diff
@@ -1853,6 +1856,32 @@ $defs: # JSON Schema definition (maybe we should move these to a seperate file)
         type: string
         default: ''
 
+  PacketDiagramConfig:
+    title: Packet Diagram Config
+    allOf: [{ $ref: '#/$defs/BaseDiagramConfig' }]
+    description: The object containing configurations specific for packet diagrams.
+    type: object
+    unevaluatedProperties: false
+    properties:
+      rowHeight:
+        description: The height of each row in the packet diagram.
+        default: 32
+      bitWidth:
+        description: The width of each bit in the packet diagram.
+        default: 32
+      bitsPerRow:
+        description: The number of bits to display per row.
+        default: 32
+      showBits:
+        description: Toggle to display or hide bit numbers.
+        default: true
```

Comment:
Maybe we should also add this to grammar similar to `pie`? It doesn't need other configs similar to `showValues` in `sankey`?

Reply:
From an ease of use perspective, you're right. But I think it's better to keep the config separate. 
As this is default true, we'd have to add a `hideBits` to the grammar instead, which might be confusing. 
Packet diagrams need bits by default.